submissions/TaniaBondarenko/html-movie-seat-booking/styles.css

```diff
@@ -0,0 +1,362 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background-color: #9b9b9b;
+}
+
+.mainContainer {
```

Comment:
Add padding to the main container instead of adding `margin` to each element.

![image](https://user-images.githubusercontent.com/16196199/107158209-3d9ff680-6991-11eb-90ca-71b67021a0e6.png)

Reply:
got it